programs/models.py

```diff
@@ -588,6 +600,7 @@ class Program(models.Model):
         on_delete=models.SET_NULL,
     )
     required_programs = models.ManyToManyField("self", related_name="dependent_programs", symmetrical=False, blank=True)
+    excludes_programs = models.ManyToManyField("self", related_name="excluded_by", symmetrical=False, blank=True)
```

Comment:
My Django-ness is still fresh so I might be wrong on this but would we want this to be `symmetrical=True`?
Thinking through the line above and this excludes example this is how it works in my brain. Let me know if I'm thinking about this incorrectly.

  **required_programs (asymmetric is correct):**
  Advanced_Medicaid → requires → Basic_Medicaid  ✓
  Basic_Medicaid → requires → Advanced_Medicaid  ✗ (wrong direction)

  **excluded_programs (symmetric should be true):**
  FamilyCare → excludes → MomsAndBabies  ✓
  MomsAndBabies → excludes → FamilyCare  ✓ (both directions)

Reply:
What do you think of adding docstrings for these sorts of lines? Maybe it's because I'm new but it took me a bit to think through the scenarios.

Reply:
Thank your for catching that, Your Djangoness! `symmetrical=True` is definitely what we want. Will fix!

Reply:
Just trying to follow this part, would we want to set both in django admin? 

> Admin updates needed: Add [Illinois] Moms and Babies to FamilyCare program's excludes_programs

So would we also want to add FamilyCare to Moms and Babies excludes_programs?

The symmetric behavior is hard for me to wrap my head around, because if they both exclude the other, which is prioritized? Let me know if my Friday brain isn't making sense 😁